corporate/views/remote_billing_page.py

```diff
@@ -194,6 +199,23 @@ def remote_realm_billing_finalize_login(
         # pretty recently. (And we generally don't delete these at all.)
         raise AssertionError
 
+    try:
+        handle_customer_migration_from_server_to_realms(
+            server=remote_server, realms=get_realms_info_for_push_bouncer()
+        )
+    except Exception:  # nocoverage
+        billing_logger.exception(
+            "%s: Failed to migrate customer from server (id: %s) to realms",
+            request.path,
+            remote_server.id,
+            stack_info=True,
+        )
+        raise JsonableError(
+            _(
+                "Failed to migrate customer from server to realms. Please contact support for assistance."
+            )
```

Comment:
A JsonableError is not the right error format for this code path; we should be serving an error page since it'll be the main browser window that views it, not JS code talking to the API.

We can fix that in a follow-up though; this block doesn't have coverage and hopefully should mostly not happen.